### PR TITLE
Add support for pagefault injection in KVM driver

### PR DESCRIPTION
**Pull request by mtarral**
This PR adds support for pagefault injection using the `KVMi` patches v5.

The function used is `kvmi_inject_page_fault`

cc @mdontu, @adlazar, @tklengyel

### CODE REVIEW

File: libvmi/driver/kvm/kvm.c
Comment (adlazar) [inline libvmi/driver/kvm/kvm.c:619]: Keep in mind that this function has been replaced in v6 (`kvmi_inject_exception(..., vcpu, addr, ec, vector = 14 /*PF_VECTOR*/)`).

**Review by adlazar — APPROVED** [review]

**Comment by mtarral** [comment]
This one should be straightforward.
awaiting on @tklengyel approval to merge it.

**Comment by mtarral** [comment]
ping @tklengyel, @smaresca for  :+1:, so we can get rid of this :)

**Comment by tklengyel** [comment]
@mtarral this is not the official LibVMI repo so this doesn't need our approval for merging. That said, it's fairly straight forward PR so LGTM

**Comment by mtarral** [comment]
@tklengyel you are right that it doesn't require your approval, but in the end, the goal is to merge this in `libvmi`, so it's better if you approved 20 individual PRs, than having to review thousands of lines in a single big PR in the future.

But I will merge the small and straighforward PRs, and hen I have a doubt, i can ping you to review the implementation and make the best choice.

Thanks.

**Pull request merged**
